BaseStation/PTZCamera/Camera_Control.cs

```diff
@@ -0,0 +1,81 @@
+﻿using OpenTK;
+using OpenTK.Input;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Net.Http;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace HuskyRobotics
+{
+    /// <summary>
+    /// This class is just used to test the camera controls.
+    /// </summary>
+    class Camera_Control
+    {
+        static void Main(string[] args)
```

Comment:
Make this integrated with the rest of the project. This should be called from StartBaseStation.cs